src/sql-editor/standardSql/suggestionsKindRegistry.ts

```diff
@@ -104,12 +107,15 @@ export const initSuggestionsKindRegistry = (): SuggestionKindRegistyItem[] => {
     {
       id: StatementPosition.AfterOrderByKeywords,
       name: StatementPosition.AfterOrderByKeywords,
-      kind: [SuggestionKind.Columns],
+      kind: [SuggestionKind.Columns, SuggestionKind.GroupMacro],
```

Comment:
Having trouble with the group macros not being suggested here, the resulting `SuggestionsRegistyItem` is undefined. 

Also, after Group By statement the macros are not suggested.

Reply:
I apparently forgot to add the definition for the FilterMacro and GroupMacro to the registry. Fixed now